tests/selenium/single-page-thunderbird-paypal-test.js

```diff
@@ -17,13 +17,13 @@ module.exports = function(driver, By, done) {
   driver.wait(function() {
     return driver.getCurrentUrl().then(function(url) {
       var result = true;
-      if (url.indexOf('http://localhost:3000/en-US/thunderbird/thank-you/') === -1) {
+      if (url.indexOf('http://localhost:3000/en-CA/thunderbird/thank-you/') === -1) {
```

Comment:
My best guess is this never really passed. We would just close the test manually and run the next manually.